modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java

```diff
@@ -301,6 +303,16 @@ private void _testAddObjectRelationship(String type) throws Exception {
 				objectFieldNamePrefix +
 					_objectDefinition1.getPKObjectFieldName()));
 
+		ObjectFieldSetting objectFieldSetting =
+			_objectFieldSettingLocalService.fetchObjectFieldSetting(
+				objectRelationship.getObjectFieldId2(),
+				"objectDefinitionName1");
```

Comment:
Define the name in `ObjectFieldSettingConstants`

Reply:
next commit fix this